homeassistant/components/recorder/core.py

```diff
@@ -242,6 +243,8 @@ def __init__(
         self._dialect_name: SupportedDialect | None = None
         self.enabled = True
 
+        self.max_bind_vars = SQLITE_MAX_BIND_VARS
```

Comment:
```suggestion
        self.max_bind_vars = DEFAULT_MAX_BIND_VARS
```

?

Reply:
Looks like a needed to put a comment there. I pushed another commit to explain why `SQLITE_MAX_BIND_VARS` is used

Reply:
Nice !
Oh yeah, and I missed that it's not a modern SQLite bind.